src/smc-webapp/project_settings.cjsx

```diff
@@ -618,7 +618,7 @@ ProjectControlPanel = rclass
                 <div>
                     SSH into your project: <span style={color:'#666'}>First add your public key to <a onClick={@open_authorized_keys} href=''>~/.ssh/authorized_keys</a>, then use the following username@host:</span>
                     {# WARNING: previous use of <FormControl> here completely breaks copy on Firefox.}
-                    <pre>{"#{misc.replace_all(project_id, '-', '')}@#{host}.sagemath.com"} </pre>
+                    <pre>{"#{misc.replace_all(project_id, '-', '')}@ssh.cocalc.com"} </pre>
```

Comment:
Change this back?  This code doesn't get run on kucalc, right, so it is exactly the code that should use the @{host} stuff.